### DISCUSSION TITLE
Is Budgie usable at 2560 x 1440 resolution?

### DISCUSSION THREAD

**Discussion by Solarunit** [BuddiesOfBudgie/budgie-desktop#125] [Q&A]
Is Budgie usable at 2560 x 1440 resolution?
I know it is usable at 4K with 200% scale. 
There are no options between 100% and 200% in the Budgie Control Center for 2560 x 1440 resolution, so I have many doubts...

**Comment by JoshStrobl** [comment]
Yes it is usable at this resolution at 100% scaling. Fractional scaling is supported on specific operating systems that have that patch for Mutter, for example Manjaro and Ubuntu Budgie. I can't say for certain its usability with fractional scaling (with is more upscaling then downscaling it a bit), maybe @fossfreedom can provide some of his experience with that.

**Reply by Solarunit** [reply]
>Yes it is usable at this resolution at 100% scaling.

I think at 100% UI elements would be too small for me.

> I can't say for certain its usability with fractional scaling (with is more upscaling then downscaling it a bit)

To avoid potential (fractional) scaling issues at 1440 (on Solus), I am thinking to get 4K monitor. :thinking:

**Reply by fossfreedom** [reply]
I don't know if Solus patches mutter in the same way as Ubuntu and Manjaro does.

But I can confirm that fractional scaling works fine with caveats on Ubuntu:

1. you need to logout and login after making the scaling change - say 125% - so that all graphical elements are displayed properly
2. You need to disable the lock screen because GNOME & Budgie Screensaver is incompatible.  This will happen automatically now on Ubuntu (in a few days once the fixes migrate to the 22.04 release)
3. If you intend to run high frame rate games don't use fractional scaling because behind the scenes it is using a xrandr overlay which means more screen drawing - hence slower frame-rates are possible.

**Reply by Solarunit** [reply]
thanks!

**Comment by Solarunit** [comment]
If you use 2 Monitors: 4K and 1080p, is it possible to set different scaling for each monitor?

**Comment by fossfreedom** [comment]
With fractional scaling yes you can - each monitor can have its own scaling.

Without fractional scaling you can only set one scale for all monitors.
